src/elements/x-safe.css

```diff
@@ -439,15 +451,15 @@ nav.actions.floating-actions {
     }
 
     .floating-btn button {
+        justify-content: left;
```

Comment:
Looks broken for contacts button between breakpoints at 730px and 480px.

Reply:
... but this button is removed anyway